bot/libs/tickets/views.py

```diff
@@ -0,0 +1,317 @@
+from __future__ import annotations
+
+import asyncio
+import uuid
+from typing import TYPE_CHECKING, Optional
+
+import discord
+from libs.tickets.structs import ReservedTags, TicketThread
+from libs.utils import Embed, ErrorEmbed, RoboModal, RoboView
+
+from .utils import register_user, safe_content
+
+if TYPE_CHECKING:
+    from libs.utils.context import RoboContext
+
+    from bot.cogs.tickets import Tickets
+    from bot.rodhaj import Rodhaj
+
+
+class TicketTitleModal(RoboModal, title="Ticket Title"):
+    def __init__(self, ctx: RoboContext, ticket_cog: Tickets, *args, **kwargs):
+        super().__init__(ctx=ctx, *args, **kwargs)
+
+        self.title_input = discord.ui.TextInput(
+            label="Title",
+            style=discord.TextStyle.long,
+            placeholder="Input a title...",
+            min_length=20,
+            max_length=100,
+        )
+        self.input: Optional[str] = None
+        self.ticket_cog = ticket_cog
+        self.add_item(self.title_input)
+
+    async def on_submit(
+        self, interaction: discord.Interaction[Rodhaj]
+    ) -> Optional[str]:
+        self.input = self.title_input.value
+        self.ticket_cog.in_progress_tickets[interaction.user.id].title.set()
+        await interaction.response.send_message(
+            f"The title of the ticket is set to:\n`{self.title_input.value}`",
+            ephemeral=True,
+        )
+        return self.input
+
+
+class TicketTagsSelect(discord.ui.Select):
+    def __init__(self, tickets_cog: Tickets):
+        options = [
+            discord.SelectOption(
+                label="Question",
+                value="question",
+                description="Represents one or more question",
+                emoji=discord.PartialEmoji(name="\U00002753"),
+            ),
+            discord.SelectOption(
+                label="Serious",
+                value="serious",
+                description="Represents a serious concern or question(s)",
+                emoji=discord.PartialEmoji(name="\U0001f610"),
```

Comment:
```py
emoji=discord.PartialEmoji(name="\U0001f610"), # U+1F610 Neutral Face
```

Reply:
Fixed in commit adee012304df96d8a492c485c54dedcdd8a879bb